components/encryption/src/manager/mod.rs

```diff
@@ -162,21 +164,31 @@ impl Dicts {
         file.iv = Iv::new_ctr().as_slice().to_vec();
         file.key_id = self.key_dict.current_key_id;
         file.method = compat(method);
-        self.file_dict.files.insert(fname.to_owned(), file);
+        self.file_dict.files.insert(fname.to_owned(), file.clone());
```

Comment:
Please do not clone file, `Iv` impls `Copy`.

Reply:
The problem is the `file` is a `kvproto::protos::encryptionpb::FileInfo`, which doesn't implement `Copy` trait, and we don't have control of how protobuf generates code.

Reply:
I mean we could just copy IV.

Reply:
I see. Fixed.